numba/cuda/ufuncs.py

```diff
@@ -272,4 +272,86 @@ def np_real_atanh_impl(context, builder, sig, args):
 
     db[np.degrees] = db[np.rad2deg]
 
+    db[np.bitwise_and] = {
+        '??->?': numbers.int_and_impl,
+        'bb->b': numbers.int_and_impl,
+        'BB->B': numbers.int_and_impl,
+        'hh->h': numbers.int_and_impl,
+        'HH->H': numbers.int_and_impl,
+        'ii->i': numbers.int_and_impl,
+        'II->I': numbers.int_and_impl,
+        'll->l': numbers.int_and_impl,
+        'LL->L': numbers.int_and_impl,
+        'qq->q': numbers.int_and_impl,
+        'QQ->Q': numbers.int_and_impl,
+    }
+
+    db[np.bitwise_or] = {
+        '??->?': numbers.int_or_impl,
+        'bb->b': numbers.int_or_impl,
+        'BB->B': numbers.int_or_impl,
+        'hh->h': numbers.int_or_impl,
+        'HH->H': numbers.int_or_impl,
+        'ii->i': numbers.int_or_impl,
+        'II->I': numbers.int_or_impl,
+        'll->l': numbers.int_or_impl,
+        'LL->L': numbers.int_or_impl,
+        'qq->q': numbers.int_or_impl,
+        'QQ->Q': numbers.int_or_impl,
+    }
+
+    db[np.bitwise_xor] = {
+        '??->?': numbers.int_xor_impl,
+        'bb->b': numbers.int_xor_impl,
+        'BB->B': numbers.int_xor_impl,
+        'hh->h': numbers.int_xor_impl,
+        'HH->H': numbers.int_xor_impl,
+        'ii->i': numbers.int_xor_impl,
+        'II->I': numbers.int_xor_impl,
+        'll->l': numbers.int_xor_impl,
+        'LL->L': numbers.int_xor_impl,
+        'qq->q': numbers.int_xor_impl,
+        'QQ->Q': numbers.int_xor_impl,
+    }
+
+    db[np.invert] = {
+        'l->l': numbers.int_invert_impl,
+        'i->i': numbers.int_invert_impl,
+        '?->?': numbers.int_invert_impl,
+        'b->b': numbers.int_invert_impl,
+        'h->h': numbers.int_invert_impl,
+        'q->q': numbers.int_invert_impl,
+        'L->L': numbers.int_invert_impl,
+        'I->I': numbers.int_invert_impl,
+        'H->H': numbers.int_invert_impl,
+        'Q->Q': numbers.int_invert_impl,
+    }
+
+    db[np.bitwise_not] = db[np.invert]
+
```

Comment:
I think this is superflous, and just reassigns the same key in the dict - these two keys are the same:

```
$ python -c "import numpy as np; print(np.invert is np.bitwise_not)"
True
```

I think this can be removed.